src/reducers/SignupMain.js

```diff
@@ -13,124 +13,112 @@ const configData = {
   code: "",
   tou: "",
   is_app_loaded: false,
-  is_fetching: false,
+  //is_fetching: false,
   error: false,
   DEBUG: true,
   available_languages: {},
   eidas_url: "",
   mfa_authn_idp: ""
 };
 
-const fetchingActions = [actions.GET_SIGNUP_CONFIG, actions.SPIN];
+//const fetchingActions = [
+  //actions.GET_SIGNUP_CONFIG,
+  //actions.SPIN,
+  //actions.APP_LOADING,
+  //actions.APP_FETCHING,
+  //actions.GET_CODE_STATUS,
+  //captchaActions.POST_SIGNUP_TRYCAPTCHA,
+  //resendActions.POST_SIGNUP_RESEND_VERIFICATION,
+//];
 
-const unFetchingActions = [];
+//const unFetchingActions = [
+  //actions.APP_LOADED,
+//];
 
 let signupReducer = (state = configData, action) => {
   switch (action.type) {
     case actions.APP_LOADING:
       return {
         ...state,
-        is_fetching: true,
         is_app_loaded: false
       };
     case actions.APP_LOADED:
       return {
         ...state,
-        is_fetching: false,
         is_app_loaded: true
       };
-    case actions.APP_FETCHING:
-      return {
-        ...state,
-        is_fetching: true
-      };
     case actions.GET_CODE_STATUS:
       return {
         ...state,
         ...action.payload,
-        is_fetching: true
```

Comment:
I just want to double check that it's ok to go beyond commenting this out here and fully remove it